ChatQnA/README.md

```diff
@@ -49,6 +52,41 @@ You have the flexibility to customize these parameters according to your specifi
 export TGI_LLM_ENDPOINT="http://xxx.xxx.xxx.xxx:8080"
 ```
 
+## Environment Setup for LLM-on-Ray
+To use Intel [LLM-on-Ray](https://github.com/intel/llm-on-ray) on Habana Gaudi/Gaudi2, please follow these steps:
+
+### Launch Ray Gaudi Service
+
+#### Launch a local server instance on 1 Gaudi card:
+```bash
+bash ./serving/ray_gaudi/launch_ray_service.sh
+```
+
+For gated models such as `LLAMA-2`, you will have to pass -e HUGGING_FACE_HUB_TOKEN=\<token\> to the docker run command above with a valid Hugging Face Hub read token.
+
+Please follow this link [huggingface token](https://huggingface.co/docs/hub/security-tokens) to get the access token ans export `HUGGINGFACEHUB_API_TOKEN` environment with the token.
+
+```bash
+export HUGGINGFACEHUB_API_TOKEN=<token>
```

Comment:
This is not used in llm-on-ray. We can ask user to config `use_auth_token` in the yaml file.